client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -414,7 +446,7 @@ class SimplePaymentsDialog extends Component {
 		return (
 			<Dialog
 				isVisible={ showDialog }
-				onClose={ onClose }
+				onClose={ this.handleDialogClose }
```

Comment:
yes, and I really like it. IMO the `event` is an action`, and the method is the `action-to-take`.

if `action` happens, then `action-to-take`.

if `onClose` happens, then `handleDialogClose`.